### PR TITLE
 Admissions Trial project

### PR DESCRIPTION
**Pull request by hafedEfheij**
pull request for following  
Apply best practices in HTML code.
Use semantic HTML tags.
Use CSS selectors correctly.
Use CSS box model.
Perform a code review for yourself on a past project.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+            string.charAt(0).toUpperCase() + string.slice(1); }
+            console.log(capitalizeFirstLetter('foo')); // Foo</code
+          >
+          <ul>
+            <li></li>
+          </ul>
+        </section>
+        <section class="main-section" id="section5">
+          <header>
+            <h1>section5</h1>
```
Comment (hafedEfheij) [inline index.html:208]: I think I would be better if you use h2 for the design propose

File: index.html [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+
+    - The freeCodeCamp Team
+
+    -->
+
+    <div id="main-body">
+      <nav id="navbar">
+        <header>
+          Technical documentation of Microverse application challenges
+        </header>
```
Comment (hafedEfheij) [inline index.html:39]: Nice use  of the header, I suggest defining it  with  id or class so you can design more easily with CSS

File: babel/index.babel [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    from red to green.
+  - As you start to build out your 
+    project, when tests are failing, 
+    you should get helpful errors 
+    along the way!
+    ************/
+
+// PLEASE NOTE: Adding global style rules using the * selector, or by adding rules to body {..} or html {..}, or to all elements within body or html, i.e. h1 {..}, has the potential to pollute the test suite's CSS. Try adding: * { color: red }, for a quick example!
+
+// Once you have read the above messages, you can delete all comments.
```
Comment (hafedEfheij) [inline babel/index.babel:27]: for better readability, Please do not leave unused code or files to

File: index.html [hunk 4]
```diff
@@ -0,0 +1,249 @@
...
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+
+    <div id="main-body">
+      <nav id="navbar">
```
Comment (hafedEfheij) [inline index.html:36]: I think it would be better to wrap the nav in a semantic HTML tag

**Review by hafedEfheij — COMMENTED** [review]
STATUS: CHANGES REQUIRED 
Hello @hafedEfheij ,
I am Hafed.

Great job reaching this far While you have done a great job working on this project, there are some issues that you still need to work on to go to the next project.  
keep up the good work, You are almost there, 
360_F_360671329_ylgnpA3vttPoNPWrXTtfe6kuc7f6E0g2

**Review by bienvenuushindi — APPROVED** [review]
## Status: Approved :heavy_check_mark: :heavy_check_mark:

Hi @hafedEfheij 

Your project is complete!🥇. There is nothing else to say than  🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- Polite comments

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**